extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -4,12 +4,24 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', [
 	'wikia.log'
 ], function(DOMElementTweaker, log) {
 	'use strict';
-	var logGroup = 'ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory';
+	var logGroup = 'ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory',
+		autoPlayClassName = 'autoplay',
+		videoPlayerClassName = 'video-player',
+		rightVideoPlayerClassName = 'video-player-right';
```

Comment:
how about using one convention for class names instead of two?
you have `autoplay` as a modifier without `video-player` prefix but `video-player-right` is also used as modifier but has the `video-player` prefix. Can we use version with or without prefix in both?

Second question, `autoplay` class describes the behavior of video, `video-player-right` class describes the positioning? Can we unify the class naming convention in this file so they either represent what video does or how video looks.